shell/platform/embedder/embedder.h

```diff
@@ -645,7 +647,15 @@ FlutterEngineResult FlutterEngineDispatchSemanticsAction(
     size_t data_length);
 
 // Notify the engine that a vsync event occurred. A baton passed to the
-// platform via the vsync callback must be returned.
+// platform via the vsync callback must be returned. This call must be made on
+// the thread on which the call to |FlutterEngineRun| was made.
+//
+// |frame_start_time_nanos| is the point at which the vsync event occurred.
```

Comment:
Should we change 'occurred' to 'occured or will occur' as now it could be a future time?

Reply:
Good call. https://github.com/flutter/engine/pull/8093